src/kfactory/kcell.py

```diff
@@ -220,6 +220,10 @@ def __setitem__(
     def get(self, __key: str, default: Any | None = None) -> Any:
         return getattr(self, __key) if hasattr(self, __key) else default
 
+    def update(self, __d: dict[str, MetaData]) -> None:
+        for k, v in __d.items():
+            setattr(self, k, v)
+
```

Comment:
why not use the more advanced update I proposed in the other PR?

Reply:
because i coulnd't get mypy to pass